### PR TITLE
Enable flake8 and fix current issues

### PR DESCRIPTION
**Pull request by toabctl**

### CODE REVIEW

File: scripts/lib/libvirt/net-start.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 args = parser.parse_args()
 
+
```
Comment (jdsn) [inline scripts/lib/libvirt/net-start.py]: unrelated

**Comment by jdsn** [comment]
:+1: lgtm

**Comment by toabctl** [comment]
@aplanas can you add your +1 again, please? @MaximilianMeister opinions on that PR?

**Comment by MaximilianMeister** [comment]
@toabctl I like it :)
+1

**Comment by aplanas** [comment]
+1 (too late, sorry).  Thanks @MaximilianMeister : )
